src/pages/api/user.ts

```diff
@@ -59,7 +59,6 @@ const handler = generateMethodRoute(
     post: {
       routeHandler: async (req) => {
         const userId = validateAndSanitizeIdString(req.body.id as string);
-        validateUserHasAccessToUserViaId(req.user as SessionUser, userId);
```

Comment:
Noticed this. Not necessary to check user has access since it's an admin only route